### PR TITLE
created tools folder inside HTML & CSS folder, created a CSS tools tx…

### PR DESCRIPTION
**Pull request by yingtong0713**
…t file where it stores all useful tools (links) for writing CSS code

### CODE REVIEW

**Review by SkySails — CHANGES_REQUESTED** [review]
This is a great start! However: _Binary files_ (such as the one provided in this pull request) are not necessary for the functionality of this repository and should be left out of the commit. This may be done using a `.gitignore` file or by manually making sure not to include the file.
Also, make a habit of not including spaces in your filenames as this might cause you some serious headaches in the future!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (SkySails) [inline .gitignore:1]: Perfect!

Comment (yingtong0713) [reply .gitignore:1]: Good catch, thanks for the tips :)

**Review by SkySails — CHANGES_REQUESTED** [review]
These changes perfectly answer to my suggestions. The only thing left to do is to remove the `.DS_Store` file from your local directory and then make a commit again. By doing this you tell Git to stop keeping track of this file, leaving it out of this PR. Otherwise, the file will be included and added to the repository, despite your correct changes to the `.gitignore`!

**Comment by yingtong0713** [comment]
The commit a937c03 already has the git change that the file DS_store is no longer tracked in addition to being in .gitignore. I kept it locally but you can see in the commit  a937c03 that the file content of .DS_store are gone. (-6KB)

**Comment by SkySails** [comment]
You might be right, it's possible that I'm just thrown off by the fact that the commit states `1 addition, 0 deletions`. My interpretation was that the file was emptied but not removed, but I am probably wrong here. LGTM!

**Pull request merged**